common/djangoapps/third_party_auth/api/urls.py

```diff
@@ -0,0 +1,12 @@
+""" URL configuration for the third party auth API """
+
+from django.conf.urls import patterns, url
+
+from .views import UserView
+
+USERNAME_PATTERN = r'(?P<username>[\w.+-]+)'
```

Comment:
@bradenmacdonald not a blocker or an issue, but this `USERNAME_PATTERN` is copy-pasted at least [five times](https://github.com/edx/edx-platform/search?utf8=%E2%9C%93&q=USERNAME_PATTERN), one of which is slightly different. Maybe this is a good occasion to move it to some common place.